.docker/Dockerfile.test

```diff
@@ -29,13 +29,14 @@ RUN cp -r /app/web/profiles/contrib/govcms/tests /app/ \
 
 COPY .docker/images/test/drutiny /usr/bin/drutiny
 
+COPY .docker/images/test/phpunit.xml /app/tests/phpunit
+COPY .docker/images/test/bootstrap.php /app/tests/phpunit
```

Comment:
Minor quirk, could these files be moved to .docker/images/test/phpunit/... mostly ".docker/images/test/bootstrap.php" looks like it could be anything.

Reply:
Yeah agreed - I thought I had done that :) fixed that up now.

Reply:
**ahoy test-phpunit** throw up an error said 'Could not read "/app/tests/phpunit/phpunit.xml"'.
Adding a '/' at the end for above two lines fixes this error.
Without the ending '/', the phpunit.xml and bootstrap.php will be duplicated as a file named phpunit rather than 'phpunit/phpunit.xml' or 'phpunit/bootstrap.php'.
So the suggested changes is:

COPY .docker/images/test/phpunit/phpunit.xml /app/tests/phpunit/
COPY .docker/images/test/phpunit/bootstrap.php /app/tests/phpunit/